tests/redirects/map_globalconf.py

```diff
@@ -1139,4 +1139,11 @@
 
     # Bug 1335040
     url_test('/gigabit/{,apply/}', 'https://learning.mozilla.org/gigabit/'),
+
+    # Bug 1329931
+    url_test('/styleguide/products/firefox-os/privacy/matters',
+             'https://developer.mozilla.org/docs/Archive/Firefox_OS/'),
+
+    # Bug 1329931
+    url_test('/firefox/os/bye/bye', 'https://support.mozilla.org/products/firefox-os/'),
```

Comment:
Please make this test a bit more meaningful and change it to `url_test('/firefox/os/{,devices/}', 'https://support.mozilla.org/products/firefox-os'),` (note I removed the last `/` on the end of the destination URL, which was causing the test to fail).